common/error.cpp

```diff
@@ -48,6 +48,19 @@ std::ostream &operator<<(std::ostream &os, const Error &err) {
 	return os;
 }
 
+Error Error::FollowedBy(const Error &err) const {
```

Comment:
Because this is not Go and `FollowedBy()` makes more sense.

Reply:
Yes, that was my thinking too. Also, it is not exactly the same as `Wrap`, because `Wrap` encapsulates the whole error, allowing you to unwrap and get the code of the wrapped error(s). `FollowedBy` only applies to the message, and only the outermost code is available for inspection.

Reply:
Alright, I see. `FollowedBy` sounds a little odd to me, but I can't think of anything better either.